### PR TITLE
Task/cer 66 creation of urgent care test suite

### PR DESCRIPTION
**Pull request by jtorano-va**
Related Story: https://vasdvp.atlassian.net/browse/CER-66

Please test the image with the testvars from this PR: https://github.com/department-of-veterans-affairs/health-apis-urgent-care-deployment/pull/24

### CODE REVIEW

File: urgent-care-tests/pom.xml [hunk 1]
```diff
@@ -14,6 +14,52 @@
...
+      <artifactId>sentinel</artifactId>
+      <version>${project.parent.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.h2database</groupId>
+      <artifactId>h2</artifactId>
+      <version>${h2.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
```
Comment (dtournour-va) [inline urgent-care-tests/pom.xml]: Why does this need CDW schemas?

**Comment by dtournour-va** [comment]
Can you finish this task with the current version of the parent that we have?

**Comment by dtournour-va** [comment]
Also conflicts

File: urgent-care-tests/pom.xml [hunk 2]
```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <parent>
-    <artifactId>health-apis-parent</artifactId>
     <groupId>gov.va.api.health</groupId>
+    <artifactId>test-starter</artifactId>
```
Comment (alecbert-va) [inline urgent-care-tests/pom.xml]: Why are you pulling in test-starter exactly? I haven't used it before so I don't know what it's offering. I see that it redefines some docker packaging but I don't know what the rest of it is doing for us.

**Comment by dtournour-va** [comment]
After talking with Alec this task is "done" in that it works.  But there needs to be some way to make this Docker image work.  Right now it doesn't get the application.properties file and it fails...

This will require us to either entrypoint.sh to pull from somewhere.  Or to pass it into the container (which I haven't found out how to do on a single command without having to overwrite entrypoint.sh)

**Comment by dtournour-va** [comment]
Updated this in the description as well.  But please test with testvars from this PR:

https://github.com/department-of-veterans-affairs/health-apis-urgent-care-deployment/pull/24

File: src/scripts/trust-dev-certs.sh [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+#
+# Updates cacerts trust store in Java home to trust:
+#   The Lighthouse development certs, internal-sys-dev.
+#   The nexus.freedomstream.io certificate.
+#   The tools.health.dev-developer.va.gov certificate.
+#
+# You will probably need root access to modify the trust store.
+#
+
+remove_old_certificate () {
```
Comment (dtournour-va) [inline src/scripts/trust-dev-certs.sh]: maybe I just dont understand what this is doing.  But why do we need this script here?

Comment (dtournour-va) [reply src/scripts/trust-dev-certs.sh]: This question is about trust-dev-cert.sh in general

Comment (alecbert-va) [reply src/scripts/trust-dev-certs.sh]: This script is never used, as far as I've seen. I'm gonna delete it until someone complains that they actually really needed it for local dev (which is I think what it was originally here for)

Comment (alecbert-va) [reply src/scripts/trust-dev-certs.sh]: Turns out test-starter uses it for... something so we should look into why it's still considered important.

**Pull request merged**